IdentityCore/src/cache/ios/MSIDKeychainTokenCache.h

```diff
@@ -26,4 +26,32 @@
 
 @interface MSIDKeychainTokenCache : NSObject<MSIDTokenCacheDataSource>
 
+/*!
+ ADAL access group.
+ */
+@property (class, readonly, nonnull) NSString *adalAccessGroup;
+
+/*!
+ Application default access group.
+ */
+@property (class, readonly, nullable) NSString *appDefaultAccessGroup;
```

Comment:
where is this used?

Reply:
I removed it since we agreed to keep MSIDKeychainTokenCache public api same as it is in ADKeychainTokenCache.